packages/survey-core/src/survey.ts

```diff
@@ -4112,6 +4112,9 @@ export class SurveyModel extends SurveyElementCore
     const questions = this.getAllQuestions(true);
     const index = questions.indexOf(q);
     if(index < 0 || index === questions.length - 1) return false;
+    const key: any = {};
```

Comment:
```suggestion
    const key: Record<string, unknown> = {};
```

Reply:
It is better to be consistent. We use to have `any` for these scenarios.

Thank you,
Andrew